spec/models/solr_document_spec.rb

```diff
@@ -124,6 +124,42 @@
     end
   end
 
+  describe '#eresources_library_display_name' do
+    eresource_bus_library = { id: 30, item_display_struct:
+                [{ barcode: '9102385731231', callnumber: nil, current_location: nil,
+                   effective_location_id: '43c011f3-adeb-4c49-b357-06772aff11de',
+                   full_shelfkey: nil, home_location: 'BUS-ELECTRONIC', library: 'BUSINESS',
+                   permanent_location_code: 'BUS-ELECTRONIC', scheme: 'LC', type: 'ONLINE' }] }
+
+    context 'when it has holdings but no phsyical items' do
+      let(:solr_doc) { SolrDocument.new(eresource_bus_library) }
```

Comment:
```suggestion
    context 'when it has holdings but no physical items' do
      eresource_bus_library = { id: 30, item_display_struct:
                [{ barcode: '9102385731231', callnumber: nil, current_location: nil,
                   effective_location_id: '43c011f3-adeb-4c49-b357-06772aff11de',
                   full_shelfkey: nil, home_location: 'BUS-ELECTRONIC', library: 'BUSINESS',
                   permanent_location_code: 'BUS-ELECTRONIC', scheme: 'LC', type: 'ONLINE' }] }
      let(:solr_doc) { SolrDocument.new(eresource_bus_library) }
```

Reply:
Tried this. Broke the `when it is an eresource and has physical copies` test. I can do this and copy and paste the data, but that just makes the file longer.